js/ui/popover.d.ts

```diff
@@ -115,6 +115,12 @@ export interface dxPopoverOptions<TComponent> extends dxPopupOptions<TComponent>
      * @public
      */
     hideOnParentScroll?: boolean;
+    /**
+     * @docid
+     * @default false
+     * @public
+     */
+    preventScrollEvents?: boolean;
```

Comment:
Isn't it enough to declare it on a popup level?

Reply:
We decided that we will not publish this option. It option should be private and will used for capability return old behavior if it will require.